### PR TITLE
Display Merritt ark on ETD view page

### PR DESCRIPTION
**Pull request by supsrinivas**
**Summary**
[DIGREPO-943](https://help.library.ucsb.edu/browse/DIGREPO-943)

* Add ETD merritt_id to solr
* Display it as Merritt ARK on ETD View

**Local ETD fixture view page**
ETD View Page

### CODE REVIEW

**Comment by supsrinivas** [comment]
@no-reply after much deliberation, I have decided to ignore unsuccessful checks from CI. Alex is trying a fix [here](https://github.com/ucsblibrary/alexandria/pull/179/files)

**Comment by supsrinivas** [comment]
Paired with @dunn on this PR. Merging to master given the approval of [this MR]('https://gitlab.com/ucsb-library/handbook/merge_requests/5') which establishes 'pair-programming' as a form of code-review.

File: app/models/concerns/metadata.rb [hunk 1]
```diff
@@ -40,7 +40,9 @@ module Metadata
     end
 
     # For Merritt ARKs
-    property :merritt_id, predicate: RDF::Vocab::DC11.identifier
+    property :merritt_id, predicate: RDF::Vocab::DC11.identifier do |index|
```
Comment (no-reply) [inline app/models/concerns/metadata.rb]: Is the plan to populate this field for all objects (including newly ingested objects without a locally defined ARK?). If so :+1:, if not, we may have a gap to close the JIRA ticket.

Comment (supsrinivas) [reply app/models/concerns/metadata.rb]: This property is now limited to ETD objects only as defined [here](https://github.com/ucsblibrary/alexandria/pull/183/files#diff-deca340d3f65dbe0cf9c3b44fb37f018R37)

**Comment by supsrinivas** [comment]
Closing PR since code changes from this PR is now part of [PR 183](https://github.com/ucsblibrary/alexandria/pull/183/files).

**Pull request closed without merging**